util/fibers/dns_resolve.cc

```diff
@@ -119,17 +119,12 @@ void DnsResolveCallback(void* ares_arg, int status, int timeouts, hostent* hoste
     return set_error();
   }
 
-  if (hostent->h_addrtype != AF_INET) {
```

Comment:
I suggest keeping the whitelist of AF_INET and AF_INET6 (there are other families that we do not support)

Reply:
ping

Reply:
I already fixed it, the changediff is just different, reversed conditions